src/lib/patch.ts

```diff
@@ -0,0 +1,128 @@
+import type { Author, Comment, State } from "@app/lib/cobs";
+import type { CommitHeader } from "@app/lib/commit";
+import type { Diff } from "@app/lib/diff";
+import type { Host } from "@app/lib/api";
+
+import { Request } from "@app/lib/api";
+
+interface IPatch {
```

Comment:
For my own understanding what's the `I` prefix?

Reply:
Interface? 😅